resources/koans.clj

```diff
@@ -1,4 +1,16 @@
-[["01_equalities" {"__" [true
+[["28_specter" {"__" [true
```

Comment:
this will make the specter koans run first after `lein koan run`. The expected values here don't seem to be used (AFAIK)